util/installation/common/util.sh

```diff
@@ -260,51 +260,96 @@ function BashrcFile {
+        source_occurrences+=("$f")
+      fi
+    fi
+  done
+
+  if [ "${#source_occurrences[@]}" -gt '0' ]; then
+    EchoWarning "WARN: You may have automatically sourced thisbdm in the following file(s):"
+    for f in ${source_occurrences[@]}; do
+      EchoWarning "- $f"
+    done
+    EchoWarning "Please check as this is not advised."
+    EchoWarning "If a matching line is a false positive, append ' #IGNORE' to its end."
+  fi
+}
+
 # Print message that tells the user to reload their shell and source the environment.
 # Arguments:
 #   $1 biodynamo install directory
-function EchoFinishThisStep {
+function EchoFinishInstallation {
   if [[ $# -ne 1 ]]; then
-    echo "ERROR in EchoFinishThisStep: Wrong number of arguments"
+    echo "ERROR in EchoFinishInstallation: Wrong number of arguments"
     exit 1
   fi
 
-  local extraStepStr="To avoid this extra step do:"
-  local haveExecStr="to have it executed automatically when starting a new shell."
+  local addToConfigStr="   ${BDM_ECHO_UNDERLINE}echo \"alias thisbdm='source $1/bin/thisbdm"
+  local setQuietEnvVarStr
 
   EchoInfo "Before running BioDynaMo execute:"
   case $SHELL in
     *bash | *zsh)
-      EchoNewStep "   ${BDM_ECHO_UNDERLINE}source $1/bin/thisbdm.sh [-q | --quiet]"
-      EchoInfo "$extraStepStr"
+      EchoNewStep "   ${BDM_ECHO_UNDERLINE}source $1/bin/thisbdm.sh"
+
       case $SHELL in
-        *bash) EchoNewStep "   ${BDM_ECHO_UNDERLINE}echo \"source $1/bin/thisbdm.sh -q\" >> $(BashrcFile)" ;;
-        *zsh)  EchoNewStep "   ${BDM_ECHO_UNDERLINE}echo \"source $1/bin/thisbdm.sh -q\" >> $(ZshrcFile)" ;;
+        *bash) addToConfigStr="$addToConfigStr.sh'\" >> $(BashrcFile)" ;;
+        *zsh)  addToConfigStr="$addToConfigStr.sh'\" >> $(ZshrcFile)" ;;
       esac
-      EchoInfo "$haveExecStr"
+      setQuietEnvVarStr="'export BDM_THISBDM_QUIET=true'"
     ;;
     *fish)
       EchoNewStep "   ${BDM_ECHO_UNDERLINE}source $1/bin/thisbdm.fish"
-      EchoInfo "$extraStepStr"
-      EchoNewStep "   ${BDM_ECHO_UNDERLINE}echo \"source $1/bin/thisbdm.fish -q\" >> ""$(fish -c 'echo $__fish_config_dir')"
-      EchoInfo "$haveExecStr"
+      addToConfigStr="$addToConfigStr.fish'\" >> (fish -c 'echo "'$__fish_config_dir'"/config.fish')"
+      setQuietEnvVarStr="'set -gx BDM_THISBDM_QUIET true'"
     ;;
     *)
-      EchoNewStep "   ${BDM_ECHO_UNDERLINE}source $1/bin/thisbdm.<sh|fish> [-q | --quiet]"
+      EchoNewStep "   ${BDM_ECHO_UNDERLINE}source $1/bin/thisbdm.{sh|fish}"
       echo
-      EchoWarning "We could not detect your default shell."
+      EchoWarning "WARN: Your login shell appears to be '$SHELL'."
       EchoWarning "BioDynaMo currently supports the following shells:"
       EchoWarning "   ${BDM_ECHO_UNDERLINE}bash, zsh, and fish."
       return
     ;;
   esac
+
+  EchoInfo "For added convenience, run (in your terminal):"
+  EchoNewStep "$addToConfigStr"
+  EchoInfo "to be able to quickly source thisbdm in your shell session."
```

Comment:
This could lead to multiple `thisbdm` aliases in the config file.
Should we detect if there is already a `thisbdm` alias or should we tell the user to add/update the config file manually?

Let's be more explicit and say that instead of `source $1/bin/thisbdm.sh` the user can just type `thisbdm` if the alias is added. 
If the alias is added automatically, add instruction that tells the user to restart the terminal afterwards.